src/overlay/Peer.h

```diff
@@ -22,6 +22,10 @@ namespace stellar
 
 typedef std::shared_ptr<SCPQuorumSet> SCPQuorumSetPtr;
 
+static auto const MAX_MESSAGE_SIZE = 0x1000000;
+// max tx size is 64KB
+static const uint64_t MAX_CLASSIC_TX_SIZE_BYTES = 64000;
```

Comment:
heads up: this const and capacity configs are placeholders for now. I'll update those once I finish simulation tuning